dataplane/forwarding/fwdport/port.go

```diff
@@ -258,8 +260,8 @@ func Input(port Port, packet fwdpacket.Packet, dir fwdpb.PortAction, ctx *fwdcon
 			Increment(port, packet.Length(), fwdpb.CounterId_COUNTER_ID_RX_DROP_PACKETS, fwdpb.CounterId_COUNTER_ID_RX_DROP_OCTETS)
 			return nil
 		}
-		packet.Logf(fwdpacket.LogDebugMessage, "transmitting packet to %q", out.ID())
-		packet.Logf(fwdpacket.LogDesc, fmt.Sprintf("%v: Transmit %v", ctx.ID, out.ID()))
+		packet.Log().V(3).Info("transmitting packet", "port", out.ID())
+		packet.Log().WithValues("context", ctx.ID, "port", out.ID())
```

Comment:
Reverse these two?

Reply:
this is intentional, WithValues mutates the log sink so that subsequent logs have those values